addons/scratchblocks/userscript.js

```diff
@@ -76,25 +79,69 @@ export default async function ({ addon, global }) {
       container.appendChild(svg);
 
       element.innerHTML = "";
+      element.classList.add("rendered");
       element.appendChild(container);
-    }
+    });
   }
 
   window.scratchblocks.renderMatching = renderMatching;
 
-  renderMatching(".blockpost pre.blocks");
+  const scratchblocks = Object.freeze(window.scratchblocks);
+
+  Object.defineProperty(window, "scratchblocks", {
+    // Block other scratchblocks scripts from loading.
+    value: { ...scratchblocks, replace: () => undefined, sa: true },
+    writable: false,
+  });
+
+  await new Promise((resolve) => {
+    if (document.readyState !== "loading") {
+      resolve();
+    } else {
+      window.addEventListener("DOMContentLoaded", resolve, { once: true });
+    }
+  });
+
+  languages = await getLocales(addon);
+  const blocks = document.querySelectorAll("pre.blocks");
+  if (blocks.length > 0) {
```

Comment:
You could have used `!=` when apple pointed out `!==`

Reply:
We banned `!=` operator fyi

Reply:
it's the same thing